### ISSUE TITLE
Cannot change build variant to globalDebug

### ISSUE THREAD

**Issue by Oki-LM7** [mapbox/mapbox-android-demo#1371]
Every time I change it, the active debug automatically changes back to chinaDebug

**Comment by langsmith** [comment]
Thanks for the ticket, @Oki-LM7 . I haven't experienced or heard about this issue with the demo app.

Can you say more about what you mean when you say `active debug`? Do you mean `globalDebug`? Are you following the _exact_ instructions at https://github.com/mapbox/mapbox-android-demo#product-flavors? 

Try invalidate cache + restart of Android Studio?: https://www.jetbrains.com/help/rider/Cleaning_System_Cache.html

**Comment by Oki-LM7** [comment]
Yes, I followed those exact instructions. I have invalidated the cache many times.  For the demo module, it gives 4 options for the active variant. ChinaDebug,ChinaRelease ,globaDebug and globalRelease. Regardless of which I change it to it will always return back to ChinaDebug.

![image](https://user-images.githubusercontent.com/49662418/91865265-71326400-ec2e-11ea-8ce4-d710eea441c1.png)


![image](https://user-images.githubusercontent.com/49662418/91865007-27497e00-ec2e-11ea-9643-77d986756c4b.png)
